vsintegration/src/vs/FsPkgs/FSharp.Project/Common.Source.CSharp/Project/FileNode.cs

```diff
@@ -1122,47 +1067,7 @@ private void RenameCaseOnlyChange(string newFileName)
 
         #endregion
 
-#region SingleFileGenerator Support methods
```

Comment:
wip features inside of directives can be moved to a feature branch.

I can:

- create a branch singleFileGenerator
- revert this commit in stage area
- remove the `#if` `#endif`
- commit 

now the singleFileGenerator branch contains all the code **active**, so easy diff (all code is a single commi) and rebase

If the codebase change, i dont think there is a build matrix with all the directives, maybe some are not aligned or do not compile anymore (this is the problem with conditional directives). 
If someone want to continue the work, can easly use this new branch